<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblTiempo">
       <property name="text">
        <string> Tiempo restante: 300</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblContadorPiedra">
       <property name="text">
        <string>piedra: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblContadorTijeras">
       <property name="text">
        <string>tijeras: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblContadorPapel">
       <property name="text">
        <string>papel: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPuntajeJugador">
       <property name="text">
        <string>puntajeJugador</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPuntajes">
       <property name="text">
        <string>Puntajes: Tijeras: 0, Piedra: 0, Papel: 0, Jugador: 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnIniciar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>249</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>btnIniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregarTijeras">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>249</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Tijeras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregarPiedra">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>249</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Piedra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregarPapel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>249</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Papel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIngresarJugador">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>249</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingresar Jugador</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>521</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
